packages/cli-kit/src/private/node/ui/components/AutocompletePrompt.test.tsx

```diff
@@ -436,11 +435,11 @@ describe('AutocompletePrompt', async () => {
     expect(onEnter).toHaveBeenCalledWith('fifth')
   })
 
-  test('allows selecting the first item after searching and triggering the loading state', async () => {
+  test('allows selecting the first item after searching', async () => {
     const onEnter = vi.fn()
 
     const search = async (term: string) => {
-      await new Promise((resolve) => setTimeout(resolve, 500))
+      await new Promise((resolve) => setTimeout(resolve, 300))
```

Comment:
Is this due to flakiness?

Reply:
I actually reduced flakiness of this test and so I was able to reduce the number of ms for search to perform.